### PR TITLE
Fix-up of PR #11402: Redeem the copyright holders' names for `source/gui/inputGestures.py` after its split from `source/gui/settingsDialogs.py`

### PR DESCRIPTION
**Pull request by JulienCochuyt**
### Link to issue number:

Fix-up of PR #11402

### Summary of the issue:

In PR #11402, part of `source/gui/settingsDialogs.py` has been extracted to the new `source/gui/inputGestures.py` for good reasons.
However, in the process, the name of the copyright holders for this portion of the code has not been reported to the header of the new file.

### Description of how this pull request fixes the issue:
Copy all the names from the copryright headers in `source/gui/settingsDialogs.py`.

### Code Review Checklist:

- [x] Pull Request description is up to date.
- [x] Unit tests.
- [x] System (end to end) tests.
- [x] Manual tests.
- [x] User Documentation.
- [x] Change log entry.
- [x] Context sensitive help for GUI changes.

### CODE REVIEW

File: source/gui/inputGestures.py [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # -*- coding: UTF-8 -*-
 # A part of NonVisual Desktop Access (NVDA)
-# Copyright (C) 2020 NV Access Limited
+# Copyright (C) 2013-2020 NV Access Limited, James Teh, Zahari Yurukov, Joseph Lee, Reef Turner,
+# Julien Cochuyt, Cyrille Bougot
```
Comment (seanbudd) [inline source/gui/inputGestures.py:4]:
```suggestion
# Copyright (C) 2013-2020 NV Access Limited, Zahari Yurukov, Joseph Lee, Julien Cochuyt, Cyrille Bougot
```

@feerrenrut and James Teh come under "NV Access Limited"

**Review by seanbudd — CHANGES_REQUESTED** [review]
Thanks for catching this, I've referenced this with the contributors list on settingsDialog as well and looks good. I've suggested to remove the 2 NV Access developers though.

File: source/gui/inputGestures.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (lukaszgo1) [reply source/gui/inputGestures.py:4]: > On second thought, it's worth using the whole list of contributors from settingsDialogs at the time. Unless we check the entire git blame history for the function, 

I believe @JulienCochuyt did exactly that judging  by the git blame invocation from the initial comment.

[the remark this answers is not in the document]
Comment (seanbudd) [reply source/gui/inputGestures.py:4]: git blame only shows the information of the previous commit that changed the line. Using this method, previous work someone may have done on the line would be erased if another person changed it.

**Comment by JulienCochuyt** [comment]
With admittedly low priority, I'm working on a script to try blame all the revisions of the given range, including those eventually fully superseded.

File: source/gui/inputGestures.py [hunk 2]
```diff
@@ -1,6 +1,7 @@
 # -*- coding: UTF-8 -*-
 # A part of NonVisual Desktop Access (NVDA)
-# Copyright (C) 2020 NV Access Limited
+# Copyright (C) 2013-2020 NV Access Limited, James Teh, Zahari Yurukov, Joseph Lee, Reef Turner,
```
Comment (lukaszgo1) [inline source/gui/inputGestures.py:3]: When working on #13294 I've verified that the copyright header herer is accurate, though it seems we don't include names of NV Access employees.
```suggestion
# Copyright (C) 2013-2020 NV Access Limited, Zahari Yurukov, Joseph Lee,
```

**Review by lukaszgo1 — CHANGES_REQUESTED** [review]
@seanbudd Since as part of different PR I've verified that the list of contributors is accurate here can we consider applying my suggestion below and merge this? While I agree that having completely precise copyright headers for moved code is not horribly important if we have an complete list it seems a waste not to use it and prefer to use an entire (likely also not completely accurate) header from gui\settingsDialogs.

File: source/gui/inputGestures.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (seanbudd) [reply source/gui/inputGestures.py:4]: @lukaszgo1 - see this initial suggestion. There's no need to use a script like Julien proposed, this suggestion is fine

**Review by seanbudd — APPROVED** [review]

**Pull request merged**